src/pages/collections/[collectionAddress]/activity.tsx

```diff
@@ -0,0 +1,321 @@
+      <div
+        className={clsx('col-span-2 flex items-center justify-center self-center ', {
+          '-ml-6': multipleWallets,
+        })}
+      >
+        {multipleWallets && (
+          <img src="/images/uturn.svg" className="mr-2 w-4 text-gray-300" alt="wallets" />
+        )}
+        <div className="flex flex-col">
+          <Link href={`/profiles/${a.wallets[0].address}`} passHref>
+            <a>
+              <Avatar
+                border
+                address={a.wallets[0].address}
+                data={{
+                  twitterHandle: a.wallets[0].profile?.handle,
+                  pfpUrl: a.wallets[0]?.profile?.profileImageUrlLowres,
+                }}
+              />
+            </a>
+          </Link>
+          {multipleWallets && (
+            <Link href={`/profiles/${a.wallets[1].address}`} passHref>
+              <a>
+                <Avatar
+                  border
+                  address={a.wallets[1].address}
+                  data={{
+                    twitterHandle: a.wallets[1].profile?.handle,
+                    pfpUrl: a.wallets[1]?.profile?.profileImageUrlLowres,
+                  }}
+                />
+              </a>
+            </Link>
+          )}
+        </div>
+      </div>
+      <div className="self-center">
+        <DisplaySOL amount={a.price.toNumber()} iconVariant="large" className="py-2 text-base" />
+      </div>
+      <div className="self-center text-base">{format(a.createdAt, 'en_US')}</div>
+    </article>
+  );
+};
+
+interface NFTGridProps {
+  activities: NftActivity[];
+  ctaVariant?: NoProfileVariant;
+  refetch: (
+    variables?: Partial<OperationVariables> | undefined
+  ) => Promise<ApolloQueryResult<None>>;
+  onLoadMore: (inView: boolean, entry: IntersectionObserverEntry) => Promise<void>;
+  hasMore: boolean;
+  showCollection?: boolean;
+  loading?: boolean;
+}
+
+export const NFTGrid: FC<NFTGridProps> = ({
+  activities,
+  refetch,
+  onLoadMore,
+  hasMore,
+  ctaVariant,
+  showCollection,
+  loading = false,
+}) => {
+  return (
+    <>
+      <div className="flex flex-col">
+        <header className="mb-2 grid grid-cols-8 px-4">
+          <span className="col-span-2 text-base text-gray-300 ">Item</span>
+          <span className="text-base text-gray-300">Event</span>
+          <span className="text-base text-gray-300">Marketplace</span>
+          <span className="col-span-2 flex justify-center text-base text-gray-300">Parties</span>
+          <span className="text-base text-gray-300">Amount</span>
+          <span className="text-base text-gray-300">Date</span>
+        </header>
+        {loading ? (
+          <>
+            <LoadingActivityCard />
+            <LoadingActivityCard />
+            <LoadingActivityCard />
+            <LoadingActivityCard />
+          </>
+        ) : (
+          <>
+            {activities.length === 0 ? (
+              <div className={`col-span-full`}>
```

Comment:
```suggestion
              <div className="col-span-full">
```

Only use js string when needed. This applies to all places using a static string for classNames.

Reply:
My bad.